e2e_tests/integrations/test_huggingface.py

```diff
@@ -461,3 +463,173 @@ def test_usages(self):
         assert "NeptuneCallback" in [
             type(callback).__name__ for callback in trainer.callback_handler.callbacks
         ]
+
+    def test_model_checkpoints_same(self, environment):
+        with init_run(project=environment.project, api_token=environment.user_token) as run:
+            run_id = run["sys/id"].fetch()
+            callback = NeptuneCallback(run=run, log_checkpoints="same")
+            training_args = self._trainer_default_attributes.copy()
```

Comment:
It's inconsistent with fixed tests version where we don't use `copy()` in `self._trainer_default_attributes.copy()`.

Here and in several other places.

Reply:
Done